src_firmware/src/driver.h

```diff
@@ -41,43 +32,62 @@ enum MotorDirection {
 	BACKWARD
 };
 
+/// @brief Control Mode - whether excact speed or position is controlled
+enum ControlModes {
+	SPEED,
+	POSITION
+};
+
 /// @brief struct representing current software version
 struct DriverVersion {
 	uint8_t major;
 	uint8_t minor;
 };
 
+enum ChannelNumber {
+	CH0,
+	CH1
+};
```

Comment:
we should get rid of this and not pass this enum to functions. there should be only one config with channel-pin mapping which would be used at initialisation. Can be done after this PR

Reply:
And how do you want to, using shell, let the program know for which channel are we setting speed?

Reply:
Unless I am misunderstanding this comment?

Reply:
it should be parsed from BLE or shell as a numeric value and propagated in further function calls. For only one channeled driver it should be set to 0 by default